plugin/trino-pinot/src/main/java/io/trino/plugin/pinot/PinotConfig.java

```diff
@@ -64,6 +64,7 @@ public class PinotConfig
     private boolean countDistinctPushdownEnabled = true;
     private boolean proxyEnabled;
     private DataSize targetSegmentPageSize = DataSize.of(1, MEGABYTE);
+    private boolean grpcQueryEnforceMetadataException;
```

Comment:
I'm not a fan of making exception handling configurable based on a configuration. 

@elonazoulay Do you have any other idea?

Reply:
I agree, especially since it's a choice between empty results masking an exception or a query failure with detailed exception (so it can be investigated and resolved).
Is there any issue with always throwing exceptions?